### PR TITLE
MAINT: Update epochs.get_data calls

### LINKED ISSUE #323: mne future warning: get_data calls
CI fails with:

> FutureWarning: The current default of copy=False will change to copy=True in 1.7. Set the value of copy explicitly to avoid this warning

same as in:

- https://github.com/mne-tools/mne-bids/issues/1190

### PR DESCRIPTION
**Pull request by larsoner**
Closes #323

1. Replace `get_data()[:, picks]` with `get_data(picks)`
2. Add `copy=False` when relevant

### CODE REVIEW

**Review by sappelhoff — APPROVED** [review]

**Comment by larsoner** [comment]
I am going to bump max line length to 88 in anticipation of some future where `black` is used

**Comment by larsoner** [comment]
I'll cut a 0.4.3 once this is in

**Comment by larsoner** [comment]
... there are a lot of steps in https://github.com/autoreject/autoreject/wiki/How-to-make-a-release but I don't think those are needed for a 0.4.3. I'm planning just to change `__version__` in `__init__.py` in this PR, merge, then cut GH release which should then land on PyPI. Then set back to `0.5.dev3` or so.

@sappelhoff is that what you did for the last point release assuming you did it?

**Comment by sappelhoff** [comment]
Yes, I did it here: https://github.com/autoreject/autoreject/pull/303/files

I am not sure whether or not the WIKI is up to date 🤔 

I think we should cherry pick the commits into the maint/0.4 branch --> https://github.com/autoreject/autoreject/tree/maint/0.4

**Comment by larsoner** [comment]
Ahh right that makes sense! I'll revert the version number change here then

File: autoreject/utils.py [hunk 1]
```diff
@@ -15,6 +16,12 @@
 from mne.forward._field_interpolation import _setup_dots
 
 
+_GDKW = dict()
```
Comment (jasmainak) [inline autoreject/utils.py:19]: for my knowledge, what does GDKW stand for? :)

**Comment by jasmainak** [comment]
Is there a bugfix here or the two approaches to `picks` are technically equivalent? I have had reports from users about problems with `picks` and index errors ... (though I don't have the data to reproduce it)

**Comment by larsoner** [comment]
the two approaches to `picks` should be equivalent

File: autoreject/utils.py (hunk 1 above)
Comment (larsoner) [reply autoreject/utils.py:19]: get data keyword arguments

**Pull request merged**